railib/api.py

```diff
@@ -444,9 +444,10 @@ def data(self):
             result["engine_name"] = self.engine
         return result
 
-    def run(self, ctx: Context, command: str, inputs: dict = None) -> Union[dict, list]:
+    def run(self, ctx: Context, command: str, language: str, inputs: dict = None) -> Union[dict, list]:
         data = self.data
         data["query"] = command
+        data["language"] = language
```

Comment:
do we use `rel` vs `sql` or emtpy vs `sql` ?

Reply:
you can pass either rel, sql, or an empty string that will get defaulted to rel in the service